symfit/core/minimizers.py

```diff
@@ -38,14 +39,15 @@ def execute(self, **options):
 
     @property
     def initial_guesses(self):
```

Comment:
Consider using `symfit.core.support.cache` to remove the boilerplate in checking for `_initial_guesses`